README.md

```diff
@@ -1,97 +1,99 @@
-# insilico_wgs
+# insilico\_wgs
 
-You can use this to create bedfiles from a refseq-database or you can do a coverage analysis using mpileup on a collection of validation samples from the novaseq and receive a bunch of output files with various coverage stats. 
+You can use the files in this repo to :
 
-# N
+* create bed files from a refseq-database 
+* do a coverage analysis using mpileup on a collection of validation samples from the novaseq and receive a bunch of output files with various coverage stats.
 
-# create bedfile
+## Create a bedfile
 
-./create_bed.py
+To create a bed file you will need a list of reference sequences (`refseq_20190301_ncbiRefSeq`) and either a list of genes or transcripts. You can create a .txt file with your list in `insilico_panels/KG/[new cool panel]/[new cool panelname].txt` and use that. 
```

Comment:
Perhaps you could start the paragraph by saying "This corresponds to subfolder create_bedfile".